microraiden/requirements.txt

```diff
@@ -9,6 +9,7 @@ ethereum-tester
 web3==3.11.1
 gevent
 click
+requests_mock
```

Comment:
Maybe the time has come to have `requirements.txt` and `requirements-dev.txt`

Reply:
Will do.